src/Submit.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Button from './Button';
+import FormGroup from './FormGroup';
+import InputBase from './InputBase';
+
+class Submit extends InputBase {
+  renderFormGroup(children) {
+    let {bsStyle, ...other} = this.props; /* eslint no-unused-vars: 0 */
```

Comment:
I'm not seeing a bug for this, I'm presuming this bug would be against babel-eslint.